tests/integration/fixtures/mini_sentry.py

```diff
@@ -164,10 +164,26 @@ def get_project_config():
             abort(403, "relay not registered")
 
         rv = {}
-        for project_id in flask_request.json["projects"]:
-            project_config = sentry.project_configs[int(project_id)]
-            if is_trusted(relay_id, project_config):
-                rv[project_id] = project_config
+        version = flask_request.args.get("version")
+        if version in (None, "1"):
+            for project_id in flask_request.json["projects"]:
+                project_config = sentry.project_configs[int(project_id)]
+                if is_trusted(relay_id, project_config):
+                    rv[project_id] = project_config
+
+        elif version == "2":
+            for public_key in flask_request.json["publicKeys"]:
+                # We store projects by id, but need to return by key
+                for project_config in sentry.project_configs.values():
+                    for key in project_config["publicKeys"]:
+                        if not is_trusted(relay_id, project_config):
+                            continue
+
+                        if key["publicKey"] == public_key:
+                            rv[public_key] = project_config
+
+        else:
+            abort(400, "unsupported version")
```

Comment:
Throw a 500 here, this is unexpected behavior and should fail the test even if Relay gracefully recovers from a 400 for whatever reason.

Reply:
Good point, that's much better.